scripts/find-unused-fields.ts

```diff
@@ -0,0 +1,348 @@
+import chalk from "chalk"
+import * as fs from "fs"
+import * as json5 from "json5"
+import * as klawSync from "klaw-sync"
+import { flatten } from "lodash"
+import { basename, relative } from "path"
+import * as ts from "typescript"
+
+function main() {
+  if (process.argv.includes("--help") || process.argv.includes("-h")) {
+    console.log(`
```

Comment:
Just a side note, but I recommend using [dedent](https://www.npmjs.com/package/dedent) so you don't have to worry so much about the weird spacing issues introduced via the multi-line strings.